tests/testthat/test-compiler-ghpages.R

```diff
@@ -23,15 +10,15 @@ test_that("error for viz with no title", {
   viz.no.title <- viz
   viz.no.title$title <- NULL
   expect_error({
-    animint2pages(viz.no.title, "no-title",server=s)
+    animint2pages(viz.no.title, "no-title")
```

Comment:
The test passes fine on my computer without the persistent server, and I thought that you wanted to add the persistent server in order to get the test working, right?
let's see if it works on Github Actions, but if it does, then I would suggest removing this persistent server functionality (simpler implementation is easier to maintain).